src/app/(sidebar)/endpoints/[[...pages]]/page.tsx

```diff
@@ -186,6 +186,7 @@ export default function Endpoints() {
                   const parsed = JSON.parse(item);
 
                   return parsed;
+                  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
interesting. why is this necessary?

Reply:
New ESLint rules are stricter about unused vars. In this case, we need to pass in a variable to `catch()`, but I didn't want to disable this rule for other cases (turning this rule off in the eslintsrc file). I thought that disabling it just for that case would be better.